x-pack/legacy/plugins/apm/public/components/app/ServiceMap/index.tsx

```diff
@@ -63,6 +71,9 @@ export function ServiceMap({ serviceName }: ServiceMapProps) {
     <div
       style={{ height: height - parseInt(theme.gutterTypes.gutterLarge, 10) }}
       ref={ref}
+      onMouseDown={() => {
+        trackApmEvent({ metric: 'service_map_click_interaction' });
```

Comment:
Is this when opening the popover? in that case, should we be explicit about that?

```suggestion
        trackApmEvent({ metric: 'service_map_open_popover' });
```

or perhaps (in case we'll allow clicking edges in the future):

```suggestion
        trackApmEvent({ metric: 'service_map_node_click' });
```